lib/ddtrace/transport/traces.rb

```diff
@@ -123,7 +123,9 @@ def send_traces(traces)
                 return send_traces(traces)
               end
             end
-          end.force
+          end
+
+          responses = responses.to_a # Force resolution of lazy enumerator
```

Comment:
May be worth commenting here that this was causing us trouble with older JRuby versions (as otherwise it may not be entirely clear that this is very deliberate)